CHANGELOG.md

```diff
@@ -21,8 +21,23 @@ Description of the upcoming release here.
 ### Fixed
 
 - [#258](https://github.com/FuelLabs/sway-libs/pull/258) Fixes incorrect instructions on how to run tests in README and docs hub.
+- [#263](https://github.com/FuelLabs/sway-libs/pull/263) Fixes `I256`'s returned bits.
```

Comment:
Do you need two rows for these?

Reply:
Yes it's best practice when there are multiple changes in a single PR.